src/_colors.css

```diff
@@ -0,0 +1,39 @@
+:root {
```

Comment:
Whoops, meant to cut a branch for these color suggestions. Since it's out in the open now, what are your thoughts on the `blue-ocean` naming convention? To me it feels more in line with the brand. 

Also I would suggest consolidating or finding a happy medium between color less than 10% difference in hue. Do we really need `white-offset: #f7f7f7` and `white-offset-alt: fafafa`?